CHANGELOG.md

```diff
@@ -26,6 +26,11 @@ Uno Changelog
 - Marked the following build property as obsolete.
     * `Gradle.Model.Repository`
 
+#### iOS build target
+- Upgraded to latest Xcode project format.
+- Fixed build errors when linking against `GStreamer.framework`.
```

Comment:
Does this one technically belong in the Support.mm -> SupportApple.mm PR? Just asking, I'm fine with leaving it in this PR...

Reply:
That's correct, the `Support.mm -> SupportApple.mm` PR fixes the GStreamer.framework linking issues. I figured that was worth mentioning too while I was updating CHANGELOG.